SampleOffer.sol

```diff
@@ -94,18 +97,22 @@ contract SampleOffer {
             paidOut += amount;
     }
 
-    function setRewardDivisor(uint _rewardDivisor) onlyClient {
+    function setRewardDivisor(uint _rewardDivisor) onlyClient noEther {
         rewardDivisor = _rewardDivisor;
     }
 
-    function setDeploymentReward(uint _deploymentReward) onlyClient {
+    function setDeploymentReward(uint _deploymentReward) onlyClient noEther {
         deploymentReward = _deploymentReward;
     }
 
-    function updateClientAddress(DAO _newClient) onlyClient {
+    function updateClientAddress(DAO _newClient) onlyClient noEther {
         client = _newClient;
     }
 
+    function () {
```

Comment:
I agree, no donations, but even in a business contract it should be possible for the client to put more ether into it, for whatever reason (dramatic trop in ether price, extend scope, ...)